readme.md

```diff
@@ -12,17 +12,27 @@ import sys
 from curtsies import FullscreenWindow, Input, FSArray
 from curtsies.fmtfuncs import red, bold, green, on_blue, yellow
 
+def setup_content:
```

Comment:
looks like this should be removed

Reply:
Right, looks like a copy-paste error to me. It's not even valid Python (no indented line afterwards); I'm slightly surprised CI didn't catch this.
(Deleted my previous comment, I didn't see enough context and misinterpreted what you said.)

Reply:
BTW I'm still wondering whether it isn't better to remove the SPACE case and inline the function.
Teaches one less useful pattern to deal with the "reinitialize the screen" case, but this is about showcasing the library, not about programming patterns. And having a function is a bit of a distraction (just like duplicate code would be).

Reply:
I like that, yeah. I think I'll leave space for now.